src/filters/simple-filter/filter-component.js

```diff
@@ -18,8 +18,7 @@ angular.module('patternfly.filters').component('pfFilter', {
     };
 
     function filterExists (filter) {
-      var foundFilter = _.find(ctrl.config.appliedFilters, {title: filter.title, value: filter.value});
-      return foundFilter !== undefined;
+      return angular.isUndefined(_.find(ctrl.config.appliedFilters, {title: filter.title, value: filter.value}));
```

Comment:
The function is 'filterExists'. This logic should return true if the filter exists. Line 37 below changed to be filterExists from !filterExists.  The maintenance of logic is due to two opposite checks to make it back to where it was.